modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/CustomizedTableFDSView.java

```diff
@@ -63,6 +65,12 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		).build();
 	}
 
+	public Map<String, Object> getOptions() {
+		return HashMapBuilder.<String, Object>put(
+			"quickActionsEnabled", true
+		).build();
+	}
```

Comment:
Do we want quick actions to be the default behavior? If yes, we should:
1. Move this to `BaseTableFDSView`
2. Hide this behing FF

Reply:
I think we do not want quick actions enabled by default. If we do, we should visit every instance reviewing the configuration, specially the icons.
I believe is better and safer leave this _off_ by default

Reply:
OK, makes sense. We can always enable it by default later on.